tests/optimize/test_lie_algebra.py

```diff
@@ -321,11 +320,11 @@ def quant_fun():
 
     opt = LieAlgebraOptimizer(circuit=quant_fun, stepsize=0.1)
 
-    for step in range(6):
+    for _ in range(6):
         circuit, cost = opt.step_and_cost()
     circuit()
 
-    assert np.isclose(cost, -2.23, atol=1e-2)
+    assert np.isclose(cost, -2.23, atol=1e-3)
```

Comment:
It looks like this is failing 🤔 
```
   File "/home/runner/work/pennylane/pennylane/tests/optimize/test_lie_algebra.py", line 327, in test_docstring_example
    assert np.isclose(cost, -2.23, atol=1e-3)
AssertionError: assert tensor(False, requires_grad=True)
 +  where tensor(False, requires_grad=True) = <function isclose at 0x7fa790f76430>(tensor(-2.22604534, requires_grad=True), -2.23, atol=0.001)
 +    where <function isclose at 0x7fa790f76430> = np.isclose
```

Reply:
I needed to run a few more iterations and compare to more accurate ground state. Should be good now.